src/components/Pagination/Pagination.stories.mdx

```diff
@@ -110,13 +114,60 @@ const onPageChange = (currentPage: number) => {
     name="Pagination with Unchanging URL"
     args={{
       className: undefined,
-      getPageHref: undefined,
       id: "pagination-id-2",
       initialPage: 7,
-      onPageChange: undefined,
+      onPageChange: (selectedPage) => {
+        console.log(`Current page: ${selectedPage}`);
+      },
+      pageCount: 100,
+    }}
+  >
+    {(args) => <Pagination {...args} />}
+  </Story>
+</Canvas>
+
+### Forcing a Page Change Programatically
+
+Generally, the selected page will change when a user clicks on a page number or
```

Comment:
This is great but I think the last sentence should be first to make it clear that this only works for the clientside unchanging URL version of the component. Also, make a note that it can only be used in combination with the `onPageChange` function.